message_app/db/db.py

```diff
@@ -40,6 +42,10 @@ def query_db(query: str, args={}, one=False):
     cur.close()
     return (rv[0] if rv else None) if one else rv
 
+def make_dicts(cursor, row):
+    """convert the retrieved data into dictionary with key is column name"""
```

Comment:
`make_dicts` usage will be fully replaced with SQLAlchemy in the future right?

Reply:
yes, after we have ORM, it can be remove